app/views/amr_heatmap/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="page_content">
```

Comment:
`page_content` is already included on every page (due to `application.html.erb`) so no need to put it here.